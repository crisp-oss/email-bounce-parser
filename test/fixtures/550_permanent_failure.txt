This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<anton@nogew.com.mx>: host
    a29392b.sse.crocodilenetworks.com[105.24.214.85] said: 550 permanent
    failure for one or more recipients (anton@nogew.com.mx:blocked) (in
    reply to end of DATA command)
