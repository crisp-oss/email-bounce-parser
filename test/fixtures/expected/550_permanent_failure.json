{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<anton@nogew.com.mx>: host a29392b.sse.crocodilenetworks.com[105.24.214.85] said: 550 permanent failure for one or more recipients (anton@nogew.com.mx:blocked) (in reply to end of DATA command)",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<anton@nogew.com.mx>: host a29392b.sse.crocodilenetworks.com[105.24.214.85] said: 550 permanent failure for one or more recipients (anton@nogew.com.mx:blocked) (in reply to end of DATA command)"
  },
  "data": {
    "error": {
      "code": {
        "basic": "550",
        "enhanced": null
      },
      "label": "permanent failure for one or more recipients (anton@nogew.com.mx:blocked)",
      "type": "action_not_taken",
      "temporary": false,
      "permanent": true,
      "data": {
        "type": null
      }
    },
    "recipient": "anton@nogew.com.mx",
    "server": {
      "hostname": "a29392b.sse.crocodilenetworks.com",
      "ip": "105.24.214.85",
      "port": null
    },
    "command": "DATA"
  }
}